examples/commands/reload.js

```diff
@@ -0,0 +1,43 @@
+/*
+THIS IS JUST THE COMMAND IT SELF, IF YOU USE THIS EXACTLY THEN YOU WILL NEED A WAY TO LOAD THE FILE LIKE THE FOLLOWING HERE:
+https://solaris.codes/erelajs/guides/moreCommands.html#before-you-start
+
+YOU ALSO NEED TO INITIATE THE MANAGER AS SHOWN HERE:
+https://solaris.codes/erelajs/guides/basics.html#first-start
+
+Or copy the code inside the run function as its simply the message and arguments.
+*/
+
+
+const Discord = require("discord.js")
+
+module.exports = {
+    name: "reload",
+    run: async (client, message, args) => {
+
+        let embed = new Discord.MessageEmbed()
+        .setTitle("Reload")
+        .setDescription("Sorry, the `reload` command can only be executed by the Developer.")
+        .setColor("#cdf785");
+        if(message.author.id !== 'YOUR DISCORD ID') return message.channel.send(embed);
+
+        if(!args[0].toLowerCase()) return message.channel.send("Please provide a command name!")
+
+        let commandName = args[0].toLowerCase()
+
+        try {
+          
+          delete require.cache[require.resolve(`./${commandName}.js`)]
```

Comment:
Looks good to me but lets make sure the command exists or not and make sure its removed the collection too and then added again
`message.client.commands.delete(key)`;